frontend/hooks/useLayers.ts

```diff
@@ -342,27 +342,27 @@ export const useLayers = () => {
-        //   }),
-        //   description: formatMessage({
-        //     defaultMessage:
-        //       'Shows the percentage of the population expected to be affected by riverine flooding in an average year.',
-        //     id: 'nwXvcg',
-        //   }),
-        //   overview: formatMessage({
-        //     defaultMessage:
-        //       'Riverine flood risk measures the percentage of population expected to be affected by Riverine flooding in an average year, accounting for existing flood-protection standards. Flood risk is assessed using hazard (inundation caused by river overflow), exposure (population in flood zone), and vulnerability.16 The existing level of flood protection is also incorporated into the risk calculation. It is important to note that this indicator represents flood risk not in terms of maximum possible impact but rather as average annual impact. The impacts from infrequent, extreme flood years are averaged with more common, less newsworthy flood years to produce the “expected annual affected population.” Higher values indicate that a greater proportion of the population is expected to be impacted by Riverine floods on average.',
-        //     id: 'BY6k/1',
-        //   }),
-        // dataSource: 'World Resources Institute',
-        // dataSourceUrl: 'https://www.wri.org/data/aqueduct-global-maps-30-data',
-        //   specification: {},
-        // },
+        {
+          id: LAYERS.RiverineFloodRisk,
+          group: LAYER_GROUPS.PriorityLayer,
+          name: formatMessage({
+            defaultMessage: 'Riverine flood risk',
+            id: 'deHJ3+',
+          }),
+          description: formatMessage({
+            defaultMessage:
+              'Shows the percentage of the population expected to be affected by riverine flooding in an average year.',
+            id: 'nwXvcg',
+          }),
+          overview: formatMessage({
+            defaultMessage:
+              'Riverine flood risk measures the percentage of population expected to be affected by Riverine flooding in an average year, accounting for existing flood-protection standards. Flood risk is assessed using hazard (inundation caused by river overflow), exposure (population in flood zone), and vulnerability.16 The existing level of flood protection is also incorporated into the risk calculation. It is important to note that this indicator represents flood risk not in terms of maximum possible impact but rather as average annual impact. The impacts from infrequent, extreme flood years are averaged with more common, less newsworthy flood years to produce the “expected annual affected population.” Higher values indicate that a greater proportion of the population is expected to be impacted by Riverine floods on average.',
+            id: 'BY6k/1',
+          }),
+          dataSource: 'World Resources Institute',
+          dataSourceUrl: 'https://www.wri.org/data/aqueduct-global-maps-30-data',
+          specification: {},
```

Comment:
The bug tooltip bug looks fixed to me! 

Just not sure if you intended to enable this toggle on this PR. The reason why this particular layer crashes is because it hasn't been added yet (the empty specification).

Reply:
It's because the user story failed for not having this map layer. But if it's not working, I will comment it again

Reply:
From the [user story](https://vizzuality.atlassian.net/browse/LET-435) (my comment):
>Anyway, it’s not relevant whether the layers are working or not because it’s not part of this user story. It’s part of this one: https://vizzuality.atlassian.net/browse/LET-357. So as long as you can show the layer option there and its info button, we’re good.